bower.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "openmrsid",
+  "description": "A self-service management system for the OpenMRS ID single sign on identity",
+  "version": "2.0.1",
```

Comment:
Change this to 2.1 -- we will be releasing it as 2.1 -- and the current work on master is towards 2.1...after 2.1 -- we will probably work off of a develop branch and master will always be in state of stability as if we were deploying it.